build_library/toolchain_util.sh

```diff
@@ -27,6 +28,9 @@ BOARD_PROFILES["amd64-usr"]="coreos:coreos/amd64/generic"
 BOARD_CHOSTS["arm64-usr"]="aarch64-cros-linux-gnu"
 BOARD_PROFILES["arm64-usr"]="coreos:coreos/arm64/generic"
 
+BOARD_CHOSTS["ppc64le-usr"]="powerpc64le-unknown-linux-gnu"
```

Comment:
We've spend more than 3 months to test/verify this cross toolchain powerpc-cros-linux-gnu for ppc64le platform. It's proves that we can't using this cross toolchain to buiding grub for ppc64le. As you known, grub is more important for OS. So I switched to native compiler to compile all packages except app-emulation/rkt for CoreOS.  If we add rkt on ppc64le, I suggest you guys add support on it. thanks.